src/applications/debt-letters/sass/debt-letters.scss

```
@@ -1,6 +1,6 @@
 .right-heading {
-  font-family: 'Bitter';
-  border-bottom: 3px solid #0071BB;
+  font-family: "Bitter";
+  border-bottom: 3px solid #0071bb;
```

You'll get a warning if you don't use SCSS variables here ([ref](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-variables.scss#L48)):

```suggestion
  border-bottom: 3px solid $color-blue;
```